myosuite/envs/myo/walk_v0.py

```diff
@@ -6,7 +6,7 @@
 import collections
 import gym
 import numpy as np
-
+from mujoco_py import functions
```

Comment:
can you remove this dependency on `mujoco_py`?

Reply:
Just deleted this, it is not necessary for the new bindings.